server/jetstream_test.go

```diff
@@ -11500,6 +11501,53 @@ func TestJetStreamSourceBasics(t *testing.T) {
 	} else if _, _, sseq := streamAndSeq(shdr); sseq != 11 {
 		t.Fatalf("Expected header sequence of 11, got %d", sseq)
 	}
+
+	// Test Filters
+	cfg = &StreamConfig{
+		Name:    "FMS3",
+		Storage: FileStorage,
+		Sources: []*StreamSource{
+			{Name: "TEST", SubjectTransforms: []SubjectTransformConfig{{Source: "dlc", Destination: "dlc2"}, {Source: "rip", Destination: "rip2"}}},
```

Comment:
I think it would be useful to test behaviour in different scenarios:
* empty source and destination
* empty destination
* empty source
* invalid subjects

Reply:
Already tested in the subject transform tests, but added a couple of tests cases anyways.